lib/bom.js

```diff
@@ -223,6 +223,11 @@ declare var NodeFilter: {
     SHOW_DOCUMENT_FRAGMENT: number;
 };
 
+declare class CustomEvent extends Event {
+  static initCustomEvent(type: DOMString, canBubble: Boolean, cancelable: Boolean, detail: Object): CustomEvent;
```

Comment:
instead of the spec's `DOMString`, we just use `string`, and `boolean` instead of `Boolean` (the former are the literals `true` and `false`, the latter is an object boxing those values)

and like below, I think `detail` should be `any` here too